src/daria/corrections/color/colorcorrection.py

```diff
@@ -72,14 +75,17 @@ class ColorCorrection:
 
     def __init__(
         self,
-        roi: Optional[Union[tuple, np.ndarray]] = None,
+        config: Optional[Union[dict, str, Path]] = None,
+        roi: Optional[Union[tuple, np.ndarray, list]] = None,
         verbosity: bool = False,
         whitebalancing: bool = True,
     ):
         """
         Constructor of converter, setting up a priori all data needed for fast conversion.
 
         Args:
+            config (dict, str, path): config file for initialization of
+                images. Can replace roi.
```

Comment:
It sounds like it overwrites `roi`. But in fact the code looks like it is the other way around. Make this either understandable in the comment .